test/integration/features/profile.feature

```diff
@@ -2,104 +2,251 @@
+     Then stdout should match "admin-user\s*: disabled\s*P\(0\)"
+      And stdout should match "anyuid\s*: disabled\s*P\(0\)"
+      And stdout should match "registry-route\s*: enabled\s*P\(0\)"
+      And stdout should match "xpaas\s*: disabled\s*P\(0\)"
+
+  Scenario: Checking that memory value was applied on profile 'foo'
+    Given Profile "foo" has state "Running"
+     And executing "minishift profile set foo" succeeds
+     Then stdout should match "Profile\s*\'foo\'\s*set\s*as\s*active\s*profile"
+     When executing "minishift ssh -- less /proc/meminfo" succeeds
+      Then stdout should match "MemTotal:\s*4[0-1][0-9]{5}\s*kB"
+
+  Scenario: Checking that disk-size value was applied on profile 'foo'
+    Given Profile "foo" has state "Running"
+     And executing "minishift profile set foo" succeeds
+     Then stdout should match "Profile\s*\'foo\'\s*set\s*as\s*active\s*profile"
+     When executing "minishift ssh -- sudo fdisk -l | grep Disk" succeeds
+      Then stdout should match "Disk \/dev\/sda: 3[0-2]\.?[0-9]{0,2} (GB|GiB)"
+
+  Scenario: Checking that cpus value was applied on profile 'foo'
+    Given Profile "foo" has state "Running"
+     And executing "minishift profile set foo" succeeds
+      Then stdout should match "Profile\s*\'foo\'\s*set\s*as\s*active\s*profile"
+      When executing "minishift ssh -- cat /proc/cpuinfo" succeeds
+      Then stdout should match "processor\s*: 0"
+       And stdout should match "processor\s*: 1"
+       And stdout should match "processor\s*: 2"
+       And stdout should match "processor\s*: 3"
+       And stdout should not match "processor\s*: [4-9]"
+
+  Scenario: As user, I can execute command on non active running profile 'minishift' from active profile 'foo'
+    Given Profile "foo" has state "Running"
+     And executing "minishift profile set foo" succeeds
+     Then stdout should match "Profile\s*\'foo\'\s*set\s*as\s*active\s*profile"
+     When executing "minishift --profile minishift stop" succeeds
+     Then stdout should contain
+     """
+     Stopping local OpenShift cluster...
+     Cluster stopped.
      """
-     A profile name must be provided. Run `minishift profile list` for a list of existing profiles.
+     And executing "minishift --profile minishift status" succeeds
+      Then stdout should match "Minishift:\s*Stopped\nProfile:\s*minishift\nOpenShift:\s*Stopped"
+
+  Scenario: As user, I can use profile alias command 'instance' to list 'minishift' as the active profile
+    This alias instance of profile can be applied to all the commands instead of profile.
+    Here only list is demonstrated with this command.
+    When executing "minishift instance list" succeeds
+    #Only validating the running and stopped profile
+    Then stdout should match "-\s*foo\s*Running\s*\(Active\)"
+     And stdout should match "-\s*minishift\s*Stopped"
+
+  Scenario: As user, I can use profile alias command 'profiles' to list 'minishift' as the active profile
```

Comment:
What's the difference to the above?

Reply:
This is just a check whether profile alias command ```profiles``` and ```instance``` atleast works on list execution.

Reply:
I would suggest to bring this to the beginning of feature. No need to have it in the middle of the file where it breaks the logical flow, which is quite hard to follow in this feature.